website/docs/Pipelines/Whole_Genome_Germline_Single_Sample_Pipeline/README.md

```diff
@@ -6,7 +6,7 @@ sidebar_position: 1
  
 | Pipeline Version | Date Updated | Documentation Author | Questions or Feedback |
 | :----: | :---: | :----: | :--------------: |
-| [WholeGenomeGermlineSingleSample_v2.5.0](https://github.com/broadinstitute/warp/releases) | October 18, 2021 | [Elizabeth Kiernan](mailto:ekiernan@broadinstitute.org) | Please file GitHub issues in WARP or contact [Kylee Degatano](mailto:kdegatano@broadinstitute.org) |
+| [WholeGenomeGermlineSingleSample_v3.0.0](https://github.com/broadinstitute/warp/releases) | November, 2021 | [Elizabeth Kiernan](mailto:ekiernan@broadinstitute.org) | Please file GitHub issues in WARP or contact [Kylee Degatano](mailto:kdegatano@broadinstitute.org) |
```

Comment:
Can we update this link to go to the search page? If you include `WholeGenomeGermlineSingleSample` as the search term it should work

Reply:
@kbergin What am I doing wrong- I can't get it to work: https://github.com/broadinstitute/warp/releases?q=WholeGenomeGermlineSingleSample&expanded=true

I had the same problem with the Exome pipeline.

Reply:
I've filed a support ticket with github to ask.

Reply:
Oh! It works if you just link to one version- the whole list will return.